src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain;
+
+import java.util.List;
+import lotto.config.LottoGameRule;
+import lotto.exception.LottoNumberOutOfRangeException;
+
+public record LottoNumber(int number) {
+
+    public LottoNumber {
+        if (number < LottoGameRule.LOTTO_MIN_VALUE.value()
+                || number > LottoGameRule.LOTTO_MAX_VALUE.value()) {
```

Comment:
조건문 내부의 조건이 복잡해 지거나, 한 개 이상의 논리 연산자로 결합되는 등의 상황에서는 해당 조건을 메서드로 추출하는 것이 가독성이 더 좋아지는 경우가 많습니다!
```suggestion
        if (isInvalidRange(number)) {
```